server/config/settings/base.py

```diff
@@ -262,3 +262,7 @@
 CORS_ORIGIN_ALLOW_ALL = True
 
 LOGIN_REDIRECT_URL = '/'
+
+REGISTER_KEYS_PATH = str(APPS_DIR.path(".envs"))
+PUBLIC_KEY = None
+PRIVATE_KEY = None
```

Comment:
Add it to production config as well. And do not use paths out of server directory.